control-plane-operator/controllers/hostedcontrolplane/oauth/config.go

```diff
@@ -57,6 +57,8 @@ func generateOAuthConfig(ctx context.Context, client crclient.Client, namespace
 		return nil, err
 	}
 
+	oauthVolumeEnsureMasterCABundle(caBundleConfigMapExists)
```

Comment:
instead of doing this here, just use another variable of type `util.PodVolumeMounts` to have the mapping for the master ca bundle.

Reply:
And then use an if caBundle.. and a mergemap() call?  I guess I was concerned about the ordering, hence the function,  but we do know this should be called first...